src/main/webapp/index.html

```diff
@@ -28,7 +28,9 @@ <h1>Census Viz</h1>
             <option value="work">work in</option>
             <option value="moved">moved to</option>
           </select>
-          <input type="list" list="location" name="location" pattern="every US state|California|New Jersey">
+          <input type="list" list="location" name="location" id="locationlist"
+          autocomplete="off" required onkeyup="validateLocation()"
```

Comment:
I think these wrapped lines should be indented: https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

Reply:
Done, thanks!